content/SCALE/SCALETutorials/SystemSettings/UpdateSCALE.md

```diff
@@ -54,4 +54,18 @@ To manually update TrueNAS, click **Install Manual Update File** and save your c
 
 Select a temporary location to store the update file and click **Choose File**. Select the <file>.iso</file> you want to upgrade to and click **Apply Update**.
 
+## Updating Pools
+
+After updating, you may find that your storage pools and boot-pool can be updated to enable some supported and requested features are not enabled on the pool.
```

Comment:
instead of may, which grants permission, suggest using might as it is possible it could happen and changing to more active voice present tense, maybe something like this?
```suggestion
After updating, you might find that you can update your storage pools and boot-pool to enable some supported and requested features that are not enabled on the pool.
```